libs/designsystem/modal/src/modal/modal-component/modal.component.ts

```diff
@@ -134,12 +135,20 @@ export class ModalComponent implements OnChanges, AfterContentInit {
   }
 
   private handleBrowserBackButton(modal: HTMLIonModalElement) {
```

Comment:
We could consider extracting this function and share it with the controller based modal...